src/lib/kit/components/Views/MultiSelectView/MultiSelectView.tsx

```diff
@@ -9,6 +9,8 @@ import './MultiSelectView.scss';
 
 const b = block('multiselect-view');
 
+const POPOVER_PLACEMENT: PopoverProps['placement'] = ['bottom', 'top'];
```

Comment:
these changes do not apply to the current task, you should remove them, if you want to change imports and parameters, it is better to do it in a separate task